### ISSUE TITLE
good metric when aggregating data for plots ? 

### ISSUE THREAD

**Issue by Mustufain** [Software-Development-Pakistan/stackoverflow-data-analysis-for-pakistan#5]
Hi, for different analysis sometimes we require to sum or average the values. In some cases the sample size is quite small so taking sum is not very ideal. What do you guys suggest should be a good metric in case of imbalance data so some countries has higher number of respondent as compared to rest of the countries like USA and Pakistan ?

**Comment by hussnainahmed** [comment]
Percentages..???

**Comment by Mustufain** [comment]
yeah percentages seems fine to me as well as from some countries the frequency of respondent is quite low.
